pkg/utils/utils.go

```diff
@@ -32,9 +32,10 @@ const (
 
 // COMPONENT_IMAGES are thee keys are based on the "container name" + "-{image,version}"
 var COMPONENT_IMAGES = map[string]string{
-	"curator":             "CURATOR_IMAGE",
-	constants.FluentdName: constants.FluentdImageEnvVar,
-	"kibana":              "KIBANA_IMAGE",
+	"curator":                       "CURATOR_IMAGE",
+	constants.FluentdName:           constants.FluentdImageEnvVar,
+	"kibana":                        "KIBANA_IMAGE",
+	constants.LogMetricExporterName: "IMAGE_LOG_METRIC_EXPORTER",
```

Comment:
inconsistent naming. somewhere it is `LogFileMetric` somewhere it is `LogMetric`